### PR TITLE
Option to create user and vhost and set permissions

### PR DESCRIPTION
**Pull request by pawel-yesware**
@justin-yesware @mkostick

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,23 +21,28 @@ dokku plugin:install https://github.com/dokku/dokku-rabbitmq.git rabbitmq
...
+rabbitmq:logs <name> [-t], Print the most recent log(s) for this service
+rabbitmq:restart <name>, Graceful shutdown and restart of the RabbitMQ service container
+rabbitmq:info <name>, Print the connection information
+rabbitmq:list, List all RabbitMQ services
+rabbitmq:clone <name> <new-name>, NOT IMPLEMENTED
+rabbitmq:expose <name> [port], Expose a RabbitMQ service on custom port if provided (random port otherwise)
+rabbitmq:unexpose <name>, Unexpose a previously exposed RabbitMQ service
+rabbitmq:start <name>, Start a previously stopped RabbitMQ service
+rabbitmq:stop <name>, Stop a running RabbitMQ service
+rabbitmq:promote <name> <app>, Promote service <name> as RABBITMQ_URL in <app>
```
Comment (mkostick) [inline README.md]: I would keep these in the original alphabetical order, and try to recreate the vertically-aligned descriptions.  I think it looks nicer and doesn't deviate from the original repo.

**Comment by mkostick** [comment]
I don't know rabbit well enough to comment on the actual commands.  I'll leave that to @justin-yesware

File: README.md [hunk 2]
```diff
@@ -21,23 +21,28 @@ dokku plugin:install https://github.com/dokku/dokku-rabbitmq.git rabbitmq
...
-rabbitmq:link <name> <app>        Link the rabbitmq service to the app
-rabbitmq:list                     List all rabbitmq services
-rabbitmq:logs <name> [-t]         Print the most recent log(s) for this service
-rabbitmq:promote <name> <app>     Promote service <name> as RABBITMQ_URL in <app>
-rabbitmq:restart <name>           Graceful shutdown and restart of the rabbitmq service container
-rabbitmq:start <name>             Start a previously stopped rabbitmq service
-rabbitmq:stop <name>              Stop a running rabbitmq service
-rabbitmq:unexpose <name>          Unexpose a previously exposed rabbitmq service
-rabbitmq:unlink <name> <app>      Unlink the rabbitmq service from the app
+rabbitmq:clone <name> <new-name>											NOT IMPLEMENTED
```
Comment (justinmills) [inline README.md]: Why is the diff so large? Did you switch to using tabs instead of spaces? It's hard to see what was added :)

File: README.md [hunk 3]
```diff
@@ -64,6 +69,22 @@ dokku rabbitmq:create lolipop
 # get connection information as follows
 dokku rabbitmq:info lolipop
 
+# create user
+dokku rabbitmq:create-user lolipop username
```
Comment (justinmills) [inline README.md]: Shouldn't this also have a password?

**Comment by justinmills** [comment]
Couple of comments, but this looks great.

File: README.md (hunk 2 above)
Comment (pawel-yesware) [inline README.md]: TABS FTW! :)

**Comment by pawel-yesware** [comment]
Pushed fixes.

Switched to spaces from tabs
added password to create-user example

**Comment by justinmills** [comment]
Looks good.

**Pull request merged**